tests/mock_servicer.py

```diff
@@ -20,6 +20,8 @@ def Run(self, request, context):
             else:
                 for res in MockServicer.table_response(MockServicer.get_test_table()):
                     yield res
+        elif SQL == "TEST VERIFY SESSION":
```

Comment:
Maybe add comments why need this string, and how we improve this in the future.

Reply:
I added a todo comment here, will improve it with an elegant way in the future.